README.md

```diff
@@ -75,8 +75,25 @@ Installation process for the setting up Wordpress on a local environment.
       brew services start selenium-server-standalone
 
       brew install chromedriver
+      brew services start chromedriver
+
       brew install phantomjs
 
+- **Install WordPress**
+	
+	  php -S localhost:8222 -t ./public
+
+  Open `http://localhost:8222` in browser
+
+  Click "Continue" and then fill in the following fields:
+
+  	Site Title: chap-press
+  	Username: chappress
+  	Password: password (Checkbox "Confirm Password")
+  	Your Email: chappress@gmail.com
+
+  Click "Install Wordpress"
+
```

Comment:
I've automated it with:
```
wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
```
The issue I brought up to you earlier was because I accidentally ran this command in the root folder instead of the `public` folder. So it does exactly as you asked and the config works exactly as it should.